### PR TITLE
Service catalog Refactor

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 65a7a7a4d41c1add694d9f218d6910450d912812:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ce0c2aa8653de931f7cd6572d37456da44115717:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d2a30e359f7989112f7113b0d140b6fbc7f506fe:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/fixtures/service_fixtures.py [hunk 1]
```diff
@@ -34,9 +34,10 @@ def catalog():
                   description="my catalog")
     try:
         cat.create()
-        yield catalog
+        yield cat
     finally:
-        cat.delete()
+        if(cat.exists):
```
Comment (mfalesni) [inline cfme/fixtures/service_fixtures.py]: please remove the ()s

**Comment by lehinevych** [comment]
The test connected with REST API fail because of https://github.com/ManageIQ/manageiq/issues/7293

**Comment by dajoRH** [comment]
Lint report for commit 35b9b5a8050ffe7a739a9630457688cb77dfca46:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f350ea56255e303d2d8b155fb277f2846f0151bd:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**